### DISCUSSION TITLE
continue the interrupted training

### DISCUSSION THREAD

**Discussion by quzopl** [bghira/SimpleTuner#938] [Q&A]
is it possible to resume the interrupted training ? before a few updates ago(Simpletuner) , it was possible to continue the training (flux) , since a while , when trying to resume , evrytime a PEFT error pops up.

**Comment by bghira** [comment]
yes, it is possible

**Reply by quzopl** [reply]
@bghira but how ? before it worked without any problem, just change the steps in config and start train again, now every time there is the same  PEFT error after loading checkpoint

**Reply by bghira** [reply]
i will bust out the crystal ball and conjure up an image of the error :) but you have to do some work and tell us what is happening. no one can just solve it for you

**Reply by quzopl** [reply]
@bghira  here is my deploy script 

> #!/bin/bash
> 
> sudo apt -y install python3.11 python3.11-venv mc
> git clone  https://github.com/bghira/SimpleTuner.git
> cd SimpleTuner
> python3.11 -m venv .venv
> source .venv/bin/activate
> pip install -U poetry pip
> poetry install 


here is config: 
![config](https://github.com/user-attachments/assets/62b7b7fe-0784-4b68-adf9-c0c507d27c06)
here is error 
![error](https://github.com/user-attachments/assets/87d41f39-9d03-45dc-b627-f9100c7c674e)

evry time is new fresh linux system , before last updates work without any problems

**Reply by bghira** [reply]
alright, i think it's fixed on the main branch. it works again here
